packages/opencensus-exporter-jaeger/src/jaeger-driver/index.ts

```diff
@@ -0,0 +1,79 @@
+/**
+ * Copyright 2018, OpenCensus Authors *
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import {types} from '@opencensus/opencensus-core';
+import * as path from 'path';
+
+const indexPath = path.dirname(require.resolve('jaeger-client'));
+
+function requireJaeguerClientModule(nodeName: string) {
+  return require(path.join(indexPath, nodeName)).default;
+}
+
+// tslint:disable-next-line:variable-name
+export const UDPSender = requireJaeguerClientModule('reporters/udp_sender');
+// tslint:disable-next-line:variable-name
+export const Utils = requireJaeguerClientModule('util');
+// tslint:disable-next-line:variable-name
+export const ThriftUtils = requireJaeguerClientModule('thrift');
+
+export type Tag = {
+  key: string,
+  // tslint:disable-next-line:no-any
+  value: any
+};
+
+export type Process = {
+  serviceName: string,
+  tags: Tag[]
+};
+
+export type SenderCallback = (numSpans: number, err?: string) => void;
+
+/**
+ * Translate opencensus Span to Jeager Thrift Span
+ * @param span
+ */
+// tslint:disable-next-line:no-any
+export function spanToThrift(span: types.Span): any {
```

Comment:
No need to annotate return value as `any`.

Reply:
Done !